# U.S. Medical Insurance Costs

## Building Database Containing All Customers Data Stored In A Separate Dictionary For Every Row In "insurance.csv"


In [31]:
import csv
customers_data = []
with open("insurance.csv") as insurance_csv:
    csv_reader = csv.DictReader(insurance_csv)
    for row in csv_reader:
        customers_data.append(row)
        
#Defining empy lists that will be filled with categorized data 

customers_age = []
customers_sex = []
customers_bmi = []
customers_children = []
customers_smoker = []
customers_region = []
customers_charge = []

for record in customers_data:
    customers_age.append((record["age"]))
    customers_sex.append((record["sex"]))
    customers_bmi.append((record["bmi"]))
    customers_children.append((record["children"]))
    customers_smoker.append((record["smoker"]))
    customers_region.append((record["region"]))
    customers_charge.append((record["charges"]))
    
## printing out first 5 database rows 
for record in customers_data[:6]:
    print(record, "\n")
    

{'age': '19', 'sex': 'female', 'bmi': '27.9', 'children': '0', 'smoker': 'yes', 'region': 'southwest', 'charges': '16884.924'} 

{'age': '18', 'sex': 'male', 'bmi': '33.77', 'children': '1', 'smoker': 'no', 'region': 'southeast', 'charges': '1725.5523'} 

{'age': '28', 'sex': 'male', 'bmi': '33', 'children': '3', 'smoker': 'no', 'region': 'southeast', 'charges': '4449.462'} 

{'age': '33', 'sex': 'male', 'bmi': '22.705', 'children': '0', 'smoker': 'no', 'region': 'northwest', 'charges': '21984.47061'} 

{'age': '32', 'sex': 'male', 'bmi': '28.88', 'children': '0', 'smoker': 'no', 'region': 'northwest', 'charges': '3866.8552'} 

{'age': '31', 'sex': 'female', 'bmi': '25.74', 'children': '0', 'smoker': 'no', 'region': 'southeast', 'charges': '3756.6216'} 



### Indexing Database to Assign Index to Every Customer

In [32]:
index = 0
customer_data_indexed = {}
for record in customers_data:
    customer_data_indexed[index] = record
    index+=1

print(customer_data_indexed[0])
print(customer_data_indexed[10])
print(customer_data_indexed[100])
print(customer_data_indexed[1000])

{'age': '19', 'sex': 'female', 'bmi': '27.9', 'children': '0', 'smoker': 'yes', 'region': 'southwest', 'charges': '16884.924'}
{'age': '25', 'sex': 'male', 'bmi': '26.22', 'children': '0', 'smoker': 'no', 'region': 'northeast', 'charges': '2721.3208'}
{'age': '41', 'sex': 'female', 'bmi': '31.6', 'children': '0', 'smoker': 'no', 'region': 'southwest', 'charges': '6186.127'}
{'age': '30', 'sex': 'male', 'bmi': '22.99', 'children': '2', 'smoker': 'yes', 'region': 'northwest', 'charges': '17361.7661'}


### Defining Customer class to instantiate all records into customer objects "customer0", "customer1"...

In [33]:
# The __init__ constructor will accept a list of data from customer_data_indexed with .items() method, we define it as "input"
# Every Customer() object will be an instanciated with "customer"+[index] format  

class Customer:
    def __init__(self, data):
        self.age = data["age"]
        self.sex = data["sex"]
        self.bmi = data["bmi"]
        self.children = data["children"]
        self.smoker = data["smoker"]
        self.region = data["region"]
        self.charges = data["charges"]    
        
input = customer_data_indexed.items()

# Items from "input" list looks like {0 : {sex, age, bmi,..}} , Dictionary generated using Dict comprehension 

customers_dict = {"customer" + str(record[0]) : Customer(record[1]) for record in input}

# Printing out instance variables to verify correct assignment

print(customers_dict["customer1"].age)
print(customers_dict["customer1"].sex)
print(customers_dict["customer23"].age)
print(customers_dict["customer123"].sex)
print(customers_dict["customer123"].region)
print(customers_dict["customer234"].region)

18
male
34
male
northeast
northwest


### Calculating Average AGE, BMI, CHILDREN, CHARGE for Customer Database


Now we have cleaned and categorized our data enough to start some basic analysis.

We are going to calculate some average values for all our customers data

In [34]:
#Defining function that calculate the average values for items in a given list

def calculate_avg(list_of_items):
    total = 0
    for items in list_of_items:
        total+= float(items)
    return total/len(list_of_items)

#calling the function with previously defined lists 

customers_age_avg = round(calculate_avg(customers_age), 2)
customers_bmi_avg = round(calculate_avg(customers_bmi), 2)
customers_children_avg = round(calculate_avg(customers_children), 2)
customers_charge_avg = round(calculate_avg(customers_charge), 2)

### Printing Rounded Customers Averages for AGE, BMI, CHILDREN, CHARGE

In [35]:
print("Customers average age is: " + str(customers_age_avg))
print("Customers average bmi is: " + str(customers_bmi_avg))
print("Customers average children is: " + str(customers_children_avg))
print("Customers average charge is: " + str(customers_charge_avg))


Customers average age is: 39.21
Customers average bmi is: 30.66
Customers average children is: 1.09
Customers average charge is: 13270.42


### Splitting customer in : "SEX" = *Male/Female*, "CHILDREN" =*Yes/No* and "SMOKER" = *Yes/No*

In [36]:
sex_male = []
sex_female = []
yes_smokers = []
no_smokers = []
yes_children = []
no_children = []

for record in customers_data:
    if record["sex"] == "male":
        sex_male.append(record)
    else:
        sex_female.append(record)

for  record in customers_data:
    if record["smoker"] == "yes":
        yes_smokers.append(record)
    else:
        no_smokers.append(record)


for  record in customers_data:
    if int(record["children"]) > 0:
        yes_children.append(record)
    else:
        no_children.append(record)

print(sex_male[0:3])
print("\n")
#print(sex_female[0:3])
#print("\n")
print(yes_smokers[0:3])
print("\n")
#print(no_smokers[0:3])
#print("\n")
print(yes_children[0:3])
print("\n")
#print(no_children[0:3])
       
        

[{'age': '18', 'sex': 'male', 'bmi': '33.77', 'children': '1', 'smoker': 'no', 'region': 'southeast', 'charges': '1725.5523'}, {'age': '28', 'sex': 'male', 'bmi': '33', 'children': '3', 'smoker': 'no', 'region': 'southeast', 'charges': '4449.462'}, {'age': '33', 'sex': 'male', 'bmi': '22.705', 'children': '0', 'smoker': 'no', 'region': 'northwest', 'charges': '21984.47061'}]


[{'age': '19', 'sex': 'female', 'bmi': '27.9', 'children': '0', 'smoker': 'yes', 'region': 'southwest', 'charges': '16884.924'}, {'age': '62', 'sex': 'female', 'bmi': '26.29', 'children': '0', 'smoker': 'yes', 'region': 'southeast', 'charges': '27808.7251'}, {'age': '27', 'sex': 'male', 'bmi': '42.13', 'children': '0', 'smoker': 'yes', 'region': 'southeast', 'charges': '39611.7577'}]


[{'age': '18', 'sex': 'male', 'bmi': '33.77', 'children': '1', 'smoker': 'no', 'region': 'southeast', 'charges': '1725.5523'}, {'age': '28', 'sex': 'male', 'bmi': '33', 'children': '3', 'smoker': 'no', 'region': 'southeast', 'charg

### Calculating AGE & BMI Averages with New Splitted Data

In [37]:
# Defining new empty lists that will contatin new splitted data
# Customers ages for Sex, Smoker Status and Childrens
sex_male_ages = []
sex_female_ages = []
yes_smokers_ages = []
no_smokers_ages = []
yes_child_ages = []
no_child_ages = []
# Customers bmi for Sex, Smoker Status and Childrens
sex_male_bmi = []
sex_female_bmi = []
yes_smokers_bmi = []
no_smokers_bmi = []
yes_child_bmi = []
no_child_bmi = []

#  Iterating through previously splitted list to get "AGES" values and append them to correct list,
#  The lists will be then stored in a dictionary to map values

for record in sex_male:
    sex_male_ages.append(record["age"])
for record in sex_female:
    sex_female_ages.append(record["age"])
for record in yes_smokers:
    yes_smokers_ages.append(record["age"])
for record in no_smokers:
    no_smokers_ages.append(record["age"])
for record in yes_children:
    yes_child_ages.append(record["age"])
for record in no_children:
    no_child_ages.append(record["age"])

# Using aliases to abbreviate keywords lenght, SexMaleAges = SM_Ages  YesSmokerAges = YS_Ages etc.,
# Defining dictionaries that will store lists of splitted data and their averages respectively
ages_splitted_dict = {"SM_Ages":sex_male_ages, "SF_Ages":sex_female_ages, "YS_Ages": yes_smokers_ages, "NS_Ages": no_smokers_ages,"YC_Ages": yes_child_ages, "NC_Ages": no_child_ages}
ages_avg_dict = {"SM_Avg": 0, "SF_Avg" : 0, "YS_Avg" : 0, "NS_Avg" : 0, "YS_Avg" : 0, "NC_Avg" : 0}

def calculate_all_age_avg(ages_dict):
    ages_avg_dict["SM_Avg"] = calculate_avg(ages_dict["SM_Ages"])
    ages_avg_dict["SF_Avg"] = calculate_avg(ages_dict["SF_Ages"])
    ages_avg_dict["YS_Avg"] = calculate_avg(ages_dict["YS_Ages"])
    ages_avg_dict["NS_Avg"] = calculate_avg(ages_dict["NC_Ages"])
    ages_avg_dict["YC_Avg"] = calculate_avg(ages_dict["YC_Ages"])
    ages_avg_dict["NC_Avg"] = calculate_avg(ages_dict["NC_Ages"])
    
# Repeating same steps for bmis, basically copy/pasted the code above modifying lists and dicts name.

for record in sex_male:
    sex_male_bmi.append(record["bmi"])
for record in sex_female:
    sex_female_bmi.append(record["bmi"])
for record in yes_smokers:
    yes_smokers_bmi.append(record["bmi"])
for record in no_smokers:
    no_smokers_bmi.append(record["bmi"])
for record in yes_children:
    yes_child_bmi.append(record["bmi"])
for record in no_children:
    no_child_bmi.append(record["bmi"])
    
bmi_splitted_dict = {"SM_Bmis":sex_male_bmi, "SF_Bmis":sex_female_bmi, "YS_Bmis": yes_smokers_bmi, "NS_Bmis": no_smokers_bmi,"YC_Bmis": yes_child_bmi, "NC_Bmis": no_child_bmi}
bmi_avg_dict = {"SM_Avg": 0, "SF_Avg" : 0, "YS_Avg" : 0, "NS_Avg" : 0, "YS_Avg" : 0, "NC_Avg" : 0}

def calculate_all_bmi_avg(bmi_dict):
    bmi_avg_dict["SM_Avg"] = calculate_avg(bmi_dict["SM_Bmis"])
    bmi_avg_dict["SF_Avg"] = calculate_avg(bmi_dict["SF_Bmis"])
    bmi_avg_dict["YS_Avg"] = calculate_avg(bmi_dict["YS_Bmis"])
    bmi_avg_dict["NS_Avg"] = calculate_avg(bmi_dict["NC_Bmis"])
    bmi_avg_dict["YC_Avg"] = calculate_avg(bmi_dict["YC_Bmis"])
    bmi_avg_dict["NC_Avg"] = calculate_avg(bmi_dict["NC_Bmis"])

# Calling both function to creare dicts and print some values to verify
calculate_all_age_avg(ages_splitted_dict)
calculate_all_bmi_avg(bmi_splitted_dict)

print("Males average AGE is: " + str(ages_avg_dict["SM_Avg"]))
print("Females average AGE is: " + str(ages_avg_dict["SF_Avg"]))
print("Smokers average AGE is: " + str(ages_avg_dict["YS_Avg"]))
print("Non Smokers average BMI is: " + str(bmi_avg_dict["NS_Avg"]))
print("With children average BMI is: " + str(bmi_avg_dict["YC_Avg"]))
print("No child average BMI is: " + str(bmi_avg_dict["NC_Avg"]))






Males average AGE is: 38.917159763313606
Females average AGE is: 39.503021148036254
Smokers average AGE is: 38.51459854014598
Non Smokers average BMI is: 30.550296167247392
With children average BMI is: 30.748370418848165
No child average BMI is: 30.550296167247392


#### Splitting data again for 2 different parameters  "SEX" and  "SMOKER STATUS"


In [38]:
calculate_all_age_avg(ages_splitted_dict)
calculate_all_bmi_avg(bmi_splitted_dict)

print("Males average AGE is: " + str(ages_avg_dict["SM_Avg"]))
print("Females average AGE is: " + str(ages_avg_dict["SF_Avg"]))
print("Smokers average AGE is: " + str(ages_avg_dict["YS_Avg"]))
print("Non Smokers average BMI is: " + str(bmi_avg_dict["NS_Avg"]))
print("With children average BMI is: " + str(bmi_avg_dict["YC_Avg"]))
print("No child average BMI is: " + str(bmi_avg_dict["NC_Avg"]))

Males average AGE is: 38.917159763313606
Females average AGE is: 39.503021148036254
Smokers average AGE is: 38.51459854014598
Non Smokers average BMI is: 30.550296167247392
With children average BMI is: 30.748370418848165
No child average BMI is: 30.550296167247392


In [39]:
# Splitting data in Males/Females + Smoker/NonSmoker in order get 4 different categories
#"ms" = Male Smoker, "fns" = Female Non Smoker 

ms  = []
mns = []
fs  = []
fns = []

for record in customers_data:
    if record["sex"] == "male":
        if record["smoker"] == "yes":
            ms.append(record)
        else:
            mns.append(record)                
    else:
        if record["smoker"] == "yes":
            fs.append(record)
        else:
            fns.append(record)
            
# printing out to verifying correct distribution, first list should contain only males+smokers, second males+non-smoker etc.

print(ms[:3], "\n")
print(mns[:3], "\n")
print(fs[:3], "\n")
print(fns[:3], "\n")


[{'age': '27', 'sex': 'male', 'bmi': '42.13', 'children': '0', 'smoker': 'yes', 'region': 'southeast', 'charges': '39611.7577'}, {'age': '30', 'sex': 'male', 'bmi': '35.3', 'children': '0', 'smoker': 'yes', 'region': 'southwest', 'charges': '36837.467'}, {'age': '31', 'sex': 'male', 'bmi': '36.3', 'children': '2', 'smoker': 'yes', 'region': 'southwest', 'charges': '38711'}] 

[{'age': '18', 'sex': 'male', 'bmi': '33.77', 'children': '1', 'smoker': 'no', 'region': 'southeast', 'charges': '1725.5523'}, {'age': '28', 'sex': 'male', 'bmi': '33', 'children': '3', 'smoker': 'no', 'region': 'southeast', 'charges': '4449.462'}, {'age': '33', 'sex': 'male', 'bmi': '22.705', 'children': '0', 'smoker': 'no', 'region': 'northwest', 'charges': '21984.47061'}] 

[{'age': '19', 'sex': 'female', 'bmi': '27.9', 'children': '0', 'smoker': 'yes', 'region': 'southwest', 'charges': '16884.924'}, {'age': '62', 'sex': 'female', 'bmi': '26.29', 'children': '0', 'smoker': 'yes', 'region': 'southeast', 'charges

### CHARGE Analysis for SEX/SMOKERS Factors

In [40]:
# Iterating through splitted lists (ms, mns, fs, fns)  to get their CHARGE value, append it to the related list 
# Then call previously defined calculate_avg() funcion that accept the list and return average value

ms_total_charge_list = []
mns_total_charge_list = []
fs_total_charge_list = []
fns_total_charge_list = []

print("There are " + str(len(ms)) + " male smokers in our database")
for record in ms:
    ms_total_charge_list.append(round(float(record["charges"])))
    ms_charge_avg = round(calculate_avg(ms_total_charge_list), 2)
    ms_charge_sum = sum(ms_total_charge_list)
print("Their avg charge is:" + str(ms_charge_avg)+ "$")
print("Their total cumulated charge is : " + str(ms_charge_sum) + "$", "\n")

print("There are " + str(len(mns)) + " male non smokers in our database")
for record in mns:
    mns_total_charge_list.append(round(float(record["charges"])))
    mns_charge_avg = round(calculate_avg(mns_total_charge_list), 2)
    mns_charge_sum = sum(mns_total_charge_list)
print("Their avg charge is:" + str(mns_charge_avg)+ "$")
print("Their total cumulated charge is : " + str(mns_charge_sum) + "$", "\n")

print("The avg insurance cost just for being smoker if your are male is: " + str(ms_charge_avg - mns_charge_avg)+ "$")
print( "It is " + str(round(((ms_charge_avg-mns_charge_avg)/mns_charge_avg) * 100, 2)) + "% higher than for non-smokers")
total_difference_percentage_m = ((ms_charge_sum-mns_charge_sum)/mns_charge_sum)*100
print("The difference for total charges is " + str(ms_charge_sum-mns_charge_sum)+"$, so  male smoker's paying about " + str(round(total_difference_percentage_m, 2))+ "% more in total cost terms", '\n')
print("There are " + str(len(fs)) + " female smokers in our database")
for record in fs:
    fs_total_charge_list.append(round(float(record["charges"])))
    fs_charge_avg = round(calculate_avg(fs_total_charge_list), 2)
    fs_charge_sum = sum(fs_total_charge_list)
print("Their avg charge is:" + str(fs_charge_avg)+ "$")
print("Their total cumulated charge is : " + str(fs_charge_sum) + "$", "\n")
print("There are " + str(len(fns)) + " female non smokers in our database")
for record in fns:
    fns_total_charge_list.append(round(float(record["charges"])))
    fns_charge_avg = round(calculate_avg(fns_total_charge_list), 2)
    fns_charge_sum = sum(fns_total_charge_list)
print("Their avg charge is:" + str(fns_charge_avg)+ "$")
print("Their total cumulated charge is : " + str(fns_charge_sum) + "$", "\n")

print("The avg insurance cost just for being smoker if your are female is: " + str(fs_charge_avg - fns_charge_avg) + "$")
print( "It is " + str(round(((fs_charge_avg-fns_charge_avg)/fns_charge_avg) * 100, 2)) + "% higher than for non-smokers", '\n')
total_difference_percentage_f = ((fs_charge_sum-fns_charge_sum)/fns_charge_sum)*100
print("The difference for total charges is " + str(fs_charge_sum-fns_charge_sum)+"$, so female smoker's paying about " + str(round(total_difference_percentage_f, 2))+ "% less in total cost terms")

There are 159 male smokers in our database
Their avg charge is:33042.01$
Their total cumulated charge is : 5253679$ 

There are 517 male non smokers in our database
Their avg charge is:8087.18$
Their total cumulated charge is : 4181071$ 

The avg insurance cost just for being smoker if your are male is: 24954.83$
It is 308.57% higher than for non-smokers
The difference for total charges is 1072608$, so  male smoker's paying about 25.65% more in total cost terms 

There are 115 female smokers in our database
Their avg charge is:30678.98$
Their total cumulated charge is : 3528083$ 

There are 547 female non smokers in our database
Their avg charge is:8762.31$
Their total cumulated charge is : 4792982$ 

The avg insurance cost just for being smoker if your are female is: 21916.67$
It is 250.12% higher than for non-smokers 

The difference for total charges is -1264899$, so female smoker's paying about -26.39% less in total cost terms


### "SEX" & "SMOKING STATUS" Factors Analysis Results 

As you can see from above analysis results, being smoker has significant cost incidence on the total charge as expected:

*Males* pay 4 times more if smokers, *Females* 3.5 times more if smokers.

The difference seems be more significant (50% more) if you are male.

Also:
- Males who smoke are paying an higher overall total cost compared to non smokers, the same is not true for females.

- Males customers who smoke have higher average charge related to females (\~ +10%). 

- Females customers who don't smoke have an higher average charge related to males(\~ -10% ).

We will investiage this further, probably other factor as higher age or children are affecting these values.


In [46]:
# Analysys for smokers ages
ms_total_age_list = []
fs_total_age_list = []
for record in ms:
    ms_total_age_list.append(record["age"])
for record in fs:
    fs_total_age_list.append(record["age"])

ms_age_avg = calculate_avg(ms_total_age_list)
fs_age_avg = calculate_avg(fs_total_age_list)
print("Male + Smoker Age Average: " + str(ms_age_avg))
print("Female + Smoker Age Average: " + str(fs_age_avg))

# Analysis for non smokers ages

mns_total_age_list = []
fns_total_age_list = []
for record in mns:
    mns_total_age_list.append(record["age"])
for record in fns:
    fns_total_age_list.append(record["age"])

mns_age_avg = calculate_avg(mns_total_age_list)
fns_age_avg = calculate_avg(fns_total_age_list)
print("Male + Non Smoker Age Average: " + str(mns_age_avg))
print("Female + Non Smoker Age Average: " + str(fns_age_avg))

#Analysys for smokers children
ms_total_child_list = []
fs_total_child_list = []
for record in ms:
    ms_total_child_list.append(record["children"])
for record in fs:
    fs_total_child_list.append(record["children"])

ms_child_avg = calculate_avg(ms_total_child_list)
fs_child_avg = calculate_avg(fs_total_child_list)
print("Male + Smoker Children Average: " + str(ms_child_avg))
print( "Female + Smoker Children Average: " + str(fs_child_avg))

# Analysisis for non smokers children
mns_total_child_list = []
fns_total_child_list = []
for record in mns:
    mns_total_child_list.append(record["children"])
for record in fns:
    fns_total_child_list.append(record["children"])

mns_child_avg = calculate_avg(mns_total_child_list)
fns_child_avg = calculate_avg(fns_total_child_list)
print("Male + Non Smoker Children Average: "  + str(mns_child_avg))
print("Female + Non Smoker Children Average: " + str(fns_child_avg))


Male + Smoker Age Average: 38.44654088050314
Female + Smoker Age Average: 38.608695652173914
Male + Non Smoker Age Average: 39.06189555125725
Female + Non Smoker Age Average: 39.691042047531994
Male + Smoker Children Average: 1.1886792452830188
Female + Smoker Children Average: 1.008695652173913
Male + Non Smoker Children Average: 1.0928433268858802
Female + Non Smoker Children Average: 1.0877513711151736



Now we have some data to evaluate. 
Some observation we can make are:

- *Male Smokers* have similar age average as *Females* but higher children average ( \~  +20%)
- *Females Non Smokers* have almost 1 year more as age average compared to the opposite sex category (Male Non Smokers) and similar children average. 

*Male Smokers* children higher children average is surely affecting their average charge but we can assume another factor like BMI is affecting higher charge avg value for *Females Non Smokers* as they have similar values compared to *Males Non Smokers.

Let's investigate further about their BMIs:



In [50]:
# Analysis for smokers BMIs

ms_total_bmi_list = []
fs_total_bmi_list = []
for record in ms:
    ms_total_bmi_list.append(record["bmi"])
for record in fs:
    fs_total_bmi_list.append(record["bmi"])

ms_bmi_avg = calculate_avg(ms_total_bmi_list)
fs_bmi_avg = calculate_avg(fs_total_bmi_list)
print("Male + Smoker BMI Average: " + str(ms_bmi_avg))
print("Female + Smoker BMI Average: " + str(fs_bmi_avg))

# Analysis for Non Smokers BMIs

mns_total_bmi_list = []
fns_total_bmi_list = []
for record in mns:
    mns_total_bmi_list.append(record["bmi"])
for record in fs:
    fns_total_bmi_list.append(record["bmi"])

mns_bmi_avg = calculate_avg(mns_total_bmi_list)
fns_bmi_avg = calculate_avg(fns_total_bmi_list)
print("Male +  Non Smoker BMI Average: " + str(mns_bmi_avg))
print("Female + Non Smoker BMI Average: " + str(fns_bmi_avg))




Male + Smoker BMI Average: 31.504182389937107
Female + Smoker BMI Average: 29.608260869565225
Male +  Non Smoker BMI Average: 30.77058027079302
Female + Non Smoker BMI Average: 29.608260869565225


Well, our analysis for smokers is partially answering and confirming us why *Males Smokers* are paying more related to *Females*.

What about *Non Smokers* instead?

Our analisys shows that BMI average is lower for *Females*, they have less than 1 year (39.06 for Males vs  39.69 for Females) and similar children average. 

A quick Google research for the question : "Is health insurance more expensive for females? " returned me this :

"*Women typically pay between 10 to 60 percent more for health insurance than men. This is due to the fact that women go to the doctor's office more frequently than men*"


*Part of why female health care is so expensive is immediately apparent: Women's reproductive health — from pregnancy and childbirth to fertility treatments and birth control — requires them to see doctors and pay for procedures that men don't require. Those services aren't cheap. "*

SOURCE : Primary Drivers of Women's Health Costs,  https://thebenefitsguide.com/why-health-care-costs-more-for-women-and-how-employers-can-help/


We confirmed what this article is generally saying about Females generally paying more, but in our case we have Males customer who smokes who also have other factors like more children in average who affects their total charge average.